### ISSUE TITLE
configuration add should always come back with some feedback

### ISSUE THREAD

**Issue by tdiesler** [animo/aries-cli#161]
**Describe the bug**

With the default, I get ...

```
aries-cli configuration add --default
Added the default agent at .../.config/aries-cli/config.yaml.
```

There is however no feedback with ...

```
aries-cli configuration add --environment=local --agent-url=http://localhost:8031 --api-key=adminkey --token=${ACCESS_TOKEN}
```

Because I didn't add the default config, I didn't know (for a while) where the config was written to.

I could take a look at this, if you like.

**Comment by blu3beri** [comment]
Yeah would be awesome if you could pick this up @tdiesler thanks!

**Comment by blu3beri** [comment]
resolved by #166
